src/routes/docs/tooling/command-line/buckets/+page.markdoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: article
+title: Deploy buckets with CLI 2.0
+description: Efficiently deploy your Appwrite buckets using the Command-Line Tool (CLI). 
+---
+
+The Appwrite CLI allows you to configure and deploy buckets across projects. Install and initialize CLI Appwrite [here](/docs/tooling/command-line/installation#Installation).
```

Comment:
DO not use `here` to link stuff, this is bad for SEO and make bad reading experience. Combine links in a natural way inside the sentence.